src/components/IssueController.jsx

```diff
@@ -26,10 +26,22 @@ export default class IssueController extends FalcorController {
     // TODO: change hardcoded issueId
     return [
       ["issues", 55, ["pubDate"]],
-      ["issues", 55, "articles", ["off-campus", "on-campus", "commentary", "creative", "in-focus"], {to: 30}, ["title", "teaser", "issueId", "category", "slug"]],
-      ["issues", 55, "articles", ["off-campus", "on-campus", "commentary", "creative", "in-focus"], {to: 30}, "authors", {to: 10}, ["name", "slug"]],
+
+      // Request the featured article
       ["issues", 55, "featured", ["title", "teaser", "issueId", "category", "slug"]],
       ["issues", 55, "featured", "authors", {to: 10}, ["name", "slug"]],
+
+      // Request first two Editor's Picks
+      ["issues", 55, "picks", {to: 1}, ["title", "teaser", "issueId", "category", "slug"]],
+      ["issues", 55, "picks", {to: 1}, "authors", {to: 10}, ["name", "slug"]],
+
+      // Request first three Trending articles
+      ["issues", 55, "trending", {to: 2}, ["title", "issueId", "category", "slug"]],
+      ["issues", 55, "trending", {to: 2}, "authors", {to: 10}, ["name", "slug"]],
+
+      // Request all remaining articles from issue
+      ["issues", 55, "articles", ["off-campus", "on-campus", "commentary", "creative", "in-focus"], {to: 30}, ["title", "teaser", "issueId", "category", "slug"]],
```

Comment:
If we are gonna let editors move this around I really recommend using an array here instead.